### PR TITLE
Scripts and documentation for importing Protein Tissue Atlas.

### PR DESCRIPTION
**Pull request by mengzhensun**

### CODE REVIEW

File: scripts/proteinAtlas/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+```bash
+python3 generate_gene_to_uniprot_mapping.py -f proteinatlas.tsv --uniprot uniprot_list.txt --gene_to_uniprot gene_to_uniprot_list.txt 
+```
+Then upload the file uniprot_list.txt containing UniProt entries separated by space to [UniProt Retrieve/ID mapping](https://www.uniprot.org/uploadlists/) to generate the UniProt \[Entry, Entry Name] pairs and save to file [uniprot_to_dcid.tsv](https://github.com/datacommonsorg/data/blob/master/scripts/proteinAtlas/uniprot_to_dcid.tsv). The pair example is \['Q96GF1', 'RN185_HUMAN']. The UniProt Entry Name is the DCID for protein instances in Data Commons. Thus we can create the mapping from gene name to protein DCID.
+
+To generate the data MCF file and enumeration files, run:
+
+```bash
+python3 parse_protein_atlas.py --database normal_tissue.tsv -g gene_to_uniprot_list -u uniprot_to_dcid.tsv -m ProteinAtlasData.mcf --tissue_mcf HumanTissueEnum.mcf --cell_mcf HumanCellTypeEnum.mcf
```
Comment (pradh) [inline scripts/proteinAtlas/README.md:68]: gene_to_uniprot_list.txt ?

File: scripts/proteinAtlas/generate_gene_to_uniprot_mapping.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    # dcid stands for uniprot entry name
+    df_gene_uniprot = df_atlas[['Gene', 'Uniprot']].dropna()
+    gene_to_uniprot = dict(
+        zip(df_gene_uniprot['Gene'], df_gene_uniprot['Uniprot']))
+
+    gene_to_uniprot_list = {}
+    uniprot_set = set()
+    for gene in gene_to_uniprot:
+        gene_to_uniprot_list[gene] = gene_to_uniprot[gene].split(', ')
+        uniprot_set |= set(gene_to_uniprot_list[gene])
```
Comment (pradh) [inline scripts/proteinAtlas/generate_gene_to_uniprot_mapping.py:52]: curious why not: uniprot_set.add(gene_to_uniprot_list[gene])

File: scripts/proteinAtlas/parse_protein_atlas.py [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+            generate_mcf(protein_dcid, tissue, cell, expression, reliability))
+    return '\n\n'.join(mcf_list)
+
+
+def get_tissue_enum(tissue):
+    """Generate a enum instance for a tissue"""
+    name = get_class_name(tissue)
+    mcf = ('Node: dcid:' + name + '\n' + 'typeOf: dcs:HumanTissueEnum' + '\n' +
+           'name: "' + name + '"\n' + 'description: "' + tissue[0].upper() +
+           tissue[1:] + '"\n' + 'domainIncludes: dcs:HumanTissueEnum\n')
```
Comment (pradh) [inline scripts/proteinAtlas/parse_protein_atlas.py:165]: readability comment from the other PR applies. i.e., have a list and join by '\n'. Or use template.

File: scripts/proteinAtlas/parse_protein_atlas_test.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            'Low': 'ProteinExpressionLow',
+            'Medium': 'ProteinExpressionMedium',
+            'High': 'ProteinExpressionHigh'
+        }
+        reliability_map = {
+            'Enhanced': 'ProteinOccurrenceReliabilityEnhanced',
+            'Supported': 'ProteinOccurrenceReliabilitySupported',
+            'Approved': 'ProteinOccurrenceReliabilityApproved',
+            'Uncertain': 'ProteinOccurrenceReliabilityUncertain'
+        }
```
Comment (pradh) [inline scripts/proteinAtlas/parse_protein_atlas_test.py:74]: nit: you can make this a global or part of mcf_from_row so you don't need to define it multiple times and pass it into the function?

File: scripts/proteinAtlas/generate_gene_to_uniprot_mapping.py (hunk 2 above)
Comment (mengzhensun) [reply scripts/proteinAtlas/generate_gene_to_uniprot_mapping.py:52]: gene_to_uniprot_list[gene] is a list, and I am adding every unique element of the it to the uniprot_set by the set union here. set.add() cannot directly take a list.

File: scripts/proteinAtlas/parse_protein_atlas.py (hunk 3 above)
Comment (mengzhensun) [reply scripts/proteinAtlas/parse_protein_atlas.py:165]: Changed. Nice suggestion!

File: scripts/proteinAtlas/parse_protein_atlas_test.py (hunk 4 above)
Comment (mengzhensun) [reply scripts/proteinAtlas/parse_protein_atlas_test.py:74]: Great suggestion!

**Pull request merged**